### PR TITLE
Implement `CLEAR_HISTORY` Action

### PR DESCRIPTION
**Pull request by pl12133**
As requested in #38, this change provides an action for removing all items from `past` and `future`.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -227,6 +232,12 @@ export default function undoable (reducer, rawConfig = {}) {
         debugEnd()
         return res
 
+      case config.clearHistoryType:
+        res = createHistory(state.present)
+        debug('clearing history', res)
```
Comment (omnidan) [inline src/index.js]: This is a detail, but `clearing history` sounds like it hasn't been done yet, which could be confusing when debugging. Maybe change this to `cleared history`?

**Comment by omnidan** [comment]
@pl12133 Thank you so much, nicely done :ok_hand: I added a comment and will merge this after your input.

**Comment by pl12133** [comment]
@omnidan The wording "cleared history" did fit much better, so it is changed now. Should be all set :+1:

**Comment by omnidan** [comment]
Thank you :grin:

**Pull request merged**